Makefile

```diff
@@ -270,14 +272,25 @@ install-travis:
 	hack/install-tools.sh
 .PHONY: install-travis
 
-# Build RPMs only for the Linux AMD64 target
+# Build RPMs only for the Linux target matching the host architecture
 #
 # Args:
 #
 # Example:
 #   make build-rpms
 build-rpms:
+ifeq ($(ARCH),x86_64)
```

Comment:
@stevekuznetsov no objections there, but would still need a way to pass in that we only want to build the current architecture? maybe a `OS_ONLY_BUILD_CURRENT_PLATFORM` var? Or maybe treat redistributable as the exception wtih `OS_BUILD_REDISTRIBUTABLE`?